packages/next-templates/src/app/productpagina-klachtenformulier/page.tsx

```diff
@@ -0,0 +1,149 @@
+'use client';
+
+import {
+  Article,
+  BreadcrumbNav,
+  BreadcrumbNavLink,
+  ButtonLink,
+  ButtonGroup,
+  Heading1,
+  Heading2,
+  Link,
+  Paragraph,
+  Page,
+  PageContent,
+  SpotlightSection,
+  UnorderedList,
+  UnorderedListItem,
+} from '@utrecht/component-library-react';
+
+import { ExampleHeader } from '@/components/ExampleHeader/ExampleHeader';
+import { ExampleNavigation } from '@/components/ExampleNavigation/ExampleNavigation';
+import { ExampleFooter } from '@/components/ExampleFooter/ExampleFooter';
+import { IconChevronRight, IconChevronUp } from '@tabler/icons-react';
+
+import '@utrecht/design-tokens/dist/index.css';
+import '@/app/styling/css/productpagina-klachtenformulier.css';
+import '@/app/styling/css/wmebv.css';
+import '@/app/styling/css/detail-page.css';
+
+import { useEffect } from 'react';
+
+export default function home() {
+  const pageTitle = 'Klacht over de gemeente doorgeven';
+  const websiteLabel = 'gemeente Voorbeeld';
+
+  useEffect(() => {
+    if (typeof document?.title === 'string') {
+      document.title = `${pageTitle} - ${websiteLabel}`;
+    }
+  }, []);
+
+  return (
+    <Page>
+      <ExampleHeader />
+      <ExampleNavigation />
+
+      <PageContent className="voorbeeld-page-content-flex">
```

Comment:
Is deze class nodig? volgens mij doet het niks

Reply:
Deze heb ik overgenomen en heb ik erop laten staan omdat (ik dacht dat) hij styling krijgt vanuit wmebv.css:

<img width="344" alt="Scherm­afbeelding 2024-03-21 om 11 05 13" src="https://github.com/frameless/gemeentevoorbeeld.nl/assets/112861180/4cc159b6-b490-4ee1-be29-95165de55b92">

Heb de class er net even afgehaald en getest en deze lijkt (voor zover ik zie) inderdaad niets te doen. 🤷🏻‍♀️
Misschien dat PageContent styling vanuit globals meekrijgt? 

Dit is een goed voorbeeld van dat ik nog niet genoeg weet over (het toepassen van) de css. /het is mij niet duidelijk waar bepaalde styling vandaan komt en hoe het precies toegepast wordt. Heb je hier nog tips voor toevallig?

Reply:
Het is wel eenbeetje verwarrend voor mij ook😂 omdat de css ligt overal en nergens. Volgens mij als je op de class hovert kan je zien waar ie vandaan komt